### PR TITLE
fixed formatting issue

### PR DESCRIPTION
**Pull request by voroyam**
replaced the .... code brackets with ----

### CODE REVIEW

**Comment by voroyam** [comment]
This PR is identical to https://github.com/owncloud/docs/pull/2010

The later you approved - here you want some changes. Why the inconsistency?

**Comment by settermjd** [comment]
> This PR is identical to #2010
> 
> The later you approved - here you want some changes. Why the inconsistency?

Except it's not identical.

**Comment by voroyam** [comment]
I copied the content of the 10.3 page in to the 10.2 and 10.1 page.

**Comment by voroyam** [comment]
no use arguing, so I have to remove them by hand... great.

**Comment by voroyam** [comment]
praised be sublime and the mark line feature.

File: modules/user_manual/pages/files/access_webdav.adoc [hunk 1]
```diff
@@ -537,7 +536,7 @@ maintain, and to share, if it’s created in a standalone file.
 
 With the file created, make the request by running the following Curl command:
 
-[source,console]
+[source,xml]
```
Comment (settermjd) [inline modules/user_manual/pages/files/access_webdav.adoc]: This isn't XML output, it's console output. This change should be reverted.

**Comment by settermjd** [comment]
> praised be sublime and the mark line feature.

I wasn't aware of what that was, a bit of searching later, and I've found the VIM equivalent. Nice feature.

**Comment by voroyam** [comment]
Sublime is like VIM but user friendly :)

**Comment by voroyam** [comment]
all seems to be fixed now @settermjd

**Comment by settermjd** [comment]
> Sublime is like VIM but user friendly :)

Nothing wrong with VIM! :-D

**Pull request merged**